bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/Ffmpeg.java

```diff
@@ -56,12 +59,19 @@ public class Ffmpeg {
     public Ffmpeg(IpCameraHandler handle, FFmpegFormat format, String ffmpegLocation, String inputArguments,
             String input, String outArguments, String output, String username, String password) {
         this.format = format;
-        this.password = password;
+
+        try {
+            this.password = URLEncoder.encode(password, StandardCharsets.UTF_8.toString());
+        } catch (UnsupportedEncodingException ex) {
+            logger.warn("encoding password failed {} ", ex.getMessage());
+            this.password = password;
+        }
```

Comment:
This can be simplified to:
```suggestion
        this.password = URLEncoder.encode(password, StandardCharsets.UTF_8);
```